src/ReduxSweetAlert.js

```diff
@@ -2,8 +2,10 @@ import React, { Component, PropTypes } from 'react';
 import { connect } from 'react-redux';
 import SweetAlert from 'sweetalert-react';
 
+import isPlainObject from './utils/is-plain-object';
+
 const mapStateToProps = state => ({
-  sweetalert: state.sweetalert,
+  sweetalert: isPlainObject(state) ? state.sweetalert : state.toJS().sweetalert,
```

Comment:
Maybe we can check `toJS` instead of  `isPlainObject`:

```js
sweetalert: typeof state.toJS === 'function' ? state.toJS().sweetalert : state.sweetalert,
```